cinema/tests/test_movie_session_api.py

```diff
@@ -51,20 +46,14 @@ def test_get_movie_sessions(self):
         }
         self.assertEqual(movie_sessions.status_code, status.HTTP_200_OK)
         for field in movie_session:
-            self.assertEqual(
-                movie_sessions.data[0][field], movie_session[field]
-            )
+            self.assertEqual(movie_sessions.data[0][field], movie_session[field])
```

Comment:
This assertion checks that the movie session details are correctly retrieved, ensuring the GET operation works as expected.